cmake/templates/TBBConfig.cmake.in

```diff
@@ -38,8 +38,7 @@ endif()
 
 set(TBB_INTERFACE_VERSION @TBB_INTERFACE_VERSION@)
 
-get_filename_component(_tbb_root "${CMAKE_CURRENT_LIST_FILE}" PATH)
-get_filename_component(_tbb_root "${_tbb_root}" PATH)
+set(_tbb_root "@TBB_ROOT@")
```

Comment:
According to this change `_tbb_root` is set during the TBBConfig generation stage, but the previous implementation sets `_tbb_root` dynamically.
Is the generation of TBBConfig intended to be done on user system with absolute path in `@TBB_ROOT@` as a part of TBB package installation? Or the generation is supposed to be done on a system where the package is prepared?

Anyway it doesn't look suitable for TBBConfig files in [official TBB packages](https://github.com/01org/tbb/releases) (which are portable, actually).

As far as I understand a _relative_ path in `@TBB_ROOT@` won't work without `CMAKE_CURRENT_LIST_FILE` as CMake will try to find TBB files relatively to user project location in this case.